CONTRIBUTING.md

```diff
@@ -61,7 +61,7 @@ This project hosts the following modules:
 You can build the SDK using the following Gradle command:
 
 ```shell script
-./gradlew assembleLibraries
+./gradlew assembleLibrariesDebug assembleLibrariesRelease
```

Comment:
let's maybe still keep `assembleLibraries`? it is for the local run, without the agent, so we can run everything at once.

Reply:
👍